stripe-patrons-data/src/test/scala/com/gu/patrons/services/StripeSubscriptionsProcessorSpec.scala

```diff
@@ -33,7 +32,7 @@ class StripeSubscriptionsProcessorSpec extends AsyncFlatSpec with Matchers {
       identityService
         .getUserIdFromEmail(subscription.customer.email)
         .map(maybeIdentityId =>
-          SafeLogger.info(s"${subscription.customer.email} - ${maybeIdentityId.getOrElse("No Identity account")}"),
+          info(s"${subscription.customer.email} - ${maybeIdentityId.getOrElse("No Identity account")}"),
```

Comment:
this is a test - there is a scalatest built in `info` method for writing test logging